core/mondoo-azure-security.mql.yaml

```
@@ -1569,23 +1559,19 @@ queries:
         ```
 
         This script iterates over all PostgreSQL servers in the specified resource group, enforcing SSL on each one. Regularly running such scripts can help maintain compliance over time, especially in dynamic environments where new servers are frequently deployed.
-  - uid: mondoo-azure-security-ensure-that-ssl-enabled-postgresql-single
-    filters: |
-      asset.platform == "azure-postgresql-server"
-    mql: |
-      azure.subscription.postgreSql.server.properties.sslEnforcement == "Enabled"
-  - uid: mondoo-azure-security-ensure-that-ssl-enabled-postgresql-single-flexible
-    filters: |
-      asset.platform == "azure-postgresql-flexible-server"
-    mql: |
-      // TLS/SSL is enforced on the server by default.
-      azure.subscription.postgreSql.flexibleServer.configuration.where(name == 'require_secure_transport').all(value.downcase == "on")
   - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql
     title: Ensure SSL connection enabled for MySQL Database Server with the latest version
     impact: 80
-    variants:
-      - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql-single
-      - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql-single-flexible
+    filters: |
+      asset.platform == "azure-mysql-flexible-server"
+    props:
+      - uid: latestTlsVersions
+        title: The latest supported TLS versions for MySQL flexible servers
+        mql: |
+          return ["TLSv1.2", "TLSv1.3", "TLSv1.2,TLSv1.3"]
```

```suggestion
          return ["TLSv1.2", "TLSv1.3", "TLSv1.2, TLSv1.3"]
```

---

I think those are just the exact strings that Azure is returning there so we need them to present the same way. Let me see if I can get azure going to confirm that though

---

@tas50 Looking at this again should it be? ... 🤔 

```
return ["TLSv1.2", "TLSv1.3", "TLSv1.2", "TLSv1.3"]
```

---

@tas50 Yeah, I remember this.

---

Actually we need this (🙄 ):
```suggestion
          return ["TLSv1.2", "TLSv1.3", "TLSv1.2,TLSv1.3","TLSv1.3,TLSv1.2"]
```